libvmi/core.c

```diff
@@ -687,8 +687,8 @@ vmi_destroy(
         return VMI_FAILURE;
 
     vmi->shutting_down = TRUE;
-    events_destroy(vmi);
     driver_destroy(vmi);
+    events_destroy(vmi);
```

Comment:
Necessary so that the callback tables are not destroyed before being able to issue the final callbacks.